pinax/stripe/tests/settings.py

```diff
@@ -32,7 +32,7 @@
 ]
 SITE_ID = 1
 PINAX_STRIPE_PUBLIC_KEY = ""
-PINAX_STRIPE_SECRET_KEY = ""
+PINAX_STRIPE_SECRET_KEY = "sk_test_01234567890123456789abcd"
```

Comment:
This shouldn't be necessary to run tests.  Tests should never hit the Stripe API.

Reply:
It is not defined to hit stripe API, but rather to interpret the error message correctly as the error message contains a mangled version of the secret key.